build.gradle

```diff
@@ -15,8 +15,8 @@ buildscript {
 
 subprojects {
     repositories {
-        jcenter()
```

Comment:
This one is weird. From a fresh clone of the repo, I was unable to get the project to build (either through ./gradlew assemble or through Android Studio, both with the same error message:
```
* What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.                                                                                               > Could not find support-fragment.jar (com.android.support:support-fragment:27.1.1).
  Searched in the following locations:                                                                                                                                          https://jcenter.bintray.com/com/android/support/support-fragment/27.1.1/support-fragment-27.1.1.jar
```
It looks like gradle doesn't move on to the next repos in the list after not finding the package on jcenter. Moving the google repo first fixes the issue. This may be a problem with my environment, and if it is, this commit can be removed from the PR, but I wanted to include it just to potentially figure out what the issue is.

Reply:
MacOS.

FWIW, I've had to order repositories in the same way in the application where I'm using AppAuth.